src/ripples/touch-ripple.jsx

```diff
@@ -55,10 +66,19 @@ const TouchRipple = React.createClass({
       hasRipples: false,
       nextKey: 0,
       ripples: [],
+      muiTheme: this.context.muiTheme ? this.context.muiTheme : ThemeManager.getMuiTheme(DefaultRawTheme),
+    };
+  },
+
+  getChildContext() {
+    return {
+      muiTheme: this.state.muiTheme,
     };
   },
 
   start(e, isRippleTouchGenerated) {
+    const theme = this.state.muiTheme.ripple;
```

Comment:
It's already available as a property `this.props.muiTheme`. Can you remove the state logic?